bundle.yaml

```diff
@@ -6,6 +6,6651 @@ metadata:
+                            of the container blocks until the hook completes or until
+                            the termination grace period is reached. More info: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks'
+                          properties:
+                            exec:
+                              description: Exec specifies the action to take.
+                              properties:
+                                command:
+                                  description: Command is the command line to execute
+                                    inside the container, the working directory for
+                                    the command  is root ('/') in the container's
+                                    filesystem. The command is simply exec'd, it is
+                                    not run inside a shell, so traditional shell instructions
+                                    ('|', etc) won't work. To use a shell, you need
+                                    to explicitly call out to that shell. Exit status
+                                    of 0 is treated as live/healthy and non-zero is
+                                    unhealthy.
+                                  items:
+                                    type: string
+                                  type: array
+                              type: object
+                            httpGet:
+                              description: HTTPGet specifies the http request to perform.
+                              properties:
+                                host:
+                                  description: Host name to connect to, defaults to
+                                    the pod IP. You probably want to set "Host" in
+                                    httpHeaders instead.
+                                  type: string
+                                httpHeaders:
+                                  description: Custom headers to set in the request.
+                                    HTTP allows repeated headers.
+                                  items:
+                                    description: HTTPHeader describes a custom header
+                                      to be used in HTTP probes
+                                    properties:
+                                      name:
+                                        description: The header field name
+                                        type: string
+                                      value:
+                                        description: The header field value
+                                        type: string
+                                    required:
+                                    - name
+                                    - value
+                                    type: object
+                              
```

Comment:
Hi @schallert,

Thank you for making this change. I am from Databricks, I've applied the CRD change and build a docker image from this PR to test it out, however the `m3db-operator` was stuck without creating the m3db-cluster. From this bundle, I can see the m3db-operator is still pulling `quay.io/m3db/m3db-operator:v0.13.0` below? Do you get a chance to test with updated images built from this PR?

Reply:
As a heads up I'm going to land this so we can work on docs. @jnyi if you're on the M3 slack, feel free to ping me and we'll see if we can debug the issue. I'll wait until I hear from Databricks that this is working before we cut a release.

Reply:
Thanks for helping out, tested this change is compatible with kubernetes v1.22